packages/base-metaocaml-ocamlfind/base-metaocaml-ocamlfind.base/opam

```diff
@@ -0,0 +1,10 @@
+opam-version: "1"
```

Comment:
you need an `opam-version: "1.2"` here, otherwise:
```
[WARNING] install is an unknown field in /home/travis/.opam/repo/default/packages/base-metaocaml-ocamlfind/base-metaocaml-ocamlfind.base/opam: is your OPAM up-to-date ?
```